s4e-web/src/app/views/map-view/view-manager/overlay-list-modal/overlay-list-modal.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {OverlayListModalComponent} from './overlay-list-modal.component';
+import {MapModule} from '../../map.module';
+import {HttpClientTestingModule} from '@angular/common/http/testing';
+import {RouterTestingModule} from '@angular/router/testing';
+import {RemoteConfigurationTestingProvider} from '../../../../app.configuration.spec';
+
+describe('OverlayListModalComponent', () => {
+  let component: OverlayListModalComponent;
+  let fixture: ComponentFixture<OverlayListModalComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [MapModule, HttpClientTestingModule, RouterTestingModule],
+      providers: [RemoteConfigurationTestingProvider]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(OverlayListModalComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Why unit tests are missing for the overlay list?

Reply:
Because it is a kind of stub PR. Everything will have to come together after API is done.

That's also because one more feature will need to be added.
(At the time of writing this code a lot of things were undecided, so writing tests to the features which would be removed didn't seem like a good idea)